scraper/scraper.py

```diff
@@ -228,5 +240,24 @@ def centre_iterator():
         logger.warning(f"Unable to scrape doctolib centers: {e}")
 
 
+def get_last_scans(outpath_format):
+
+    outpath = outpath_format.format("info_centres")
+    last_scans = {}
+
+    try:
+        with open(outpath) as info_centres:
+
+            centres_precedents  = json.load(info_centres)
+            for _, dept in centres_precedents.items():
```

Comment:
Si tu veux ne récupérer que les valeurs d'un dict, c'est mieux d'utiliser `.values()`
```suggestion
            for dept in centres_precedents.values():
```

Reply:
Effectivement :)